route_network/src/route_network/nodes/viz_routes.py

```diff
@@ -55,27 +53,28 @@
 from visualization_msgs.msg import Marker
 from visualization_msgs.msg import MarkerArray
 
-class RouteVizNode():
+
+class RouteVizNode(Node):
 
     def __init__(self):
-        """ROS node to publish the route network graph for a GeographicMap.
         """
-        rospy.init_node('viz_routes')
+        ROS node to publish the route network graph for a GeographicMap.
+        """
+        super(RouteVizNode, self).__init__("viz_routes")
```

Comment:
I think Python3 doesn't really require this anymore

https://www.journaldev.com/15911/python-super

Reply:
Ah awesome. Fixed all the other classes as well. Thanks for taking your time to review, really appreciate it!